src/app-config.ts

```diff
@@ -8,3 +8,5 @@ import { ClueAppContentComponent } from "./clue/components/clue-app-content";
 export const AppContentComponent = ClueAppContentComponent;
 export { appIcons } from "./clue/app-icons";
 import "./clue/clue.sass";
+
+import "./register-tools";
```

Comment:
For consistency, maybe add a comment here like "register the tools built into the application".

Reply:
done